packages/console/src/hooks/use-connector-in-use.ts

```diff
@@ -16,12 +16,20 @@ const useConnectorInUse = () => {
 
       const { type, target } = connector;
 
-      if (type === ConnectorType.Social) {
+      const isSocial = type === ConnectorType.Social;
+      const isBlockchain = type === ConnectorType.Blockchain;
```

Comment:
📝 Maybe should move these between branches so they aren't all evaluated before they're needed. 🤷 readability vs super duper marginal gains